### ISSUE TITLE
turn the PRL into RE5's grenade launcher 

### ISSUE THREAD

**Issue by DarkxVoid-loop** [nipkownix/re4_tweaks#264]
### Describe the feature you'd like to see added

I wish the PLR ​​can be turned into the grenade launcher of RE5 with all its ammo explosive, flash, flame, acid, nitrogen and electricity
Thank you very much if possible or if Corious has already done something similar

**Comment by laymonage** [comment]
This is way out of the scope of this project, imo.

**Comment by Scoovy** [comment]
im like 90% sure this is impossible without any way of modifiying the source code of the game
there is simply no way of adding multiple ammo types into a gun that doesnt even have ammo

**Comment by linkthehylian** [comment]
I agree with Scoovy. You basically want the PRL to be replaced with an entirely different weapon that doesn't even exist in the original files.

A good idea for another modder, maybe. But definitely not fit for a project like this.
